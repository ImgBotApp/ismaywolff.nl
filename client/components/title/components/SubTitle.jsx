import glamorous from 'glamorous'

const SubTitle = glamorous.p({
  textAlign: 'center',
  margin: 0
})

export default SubTitle
