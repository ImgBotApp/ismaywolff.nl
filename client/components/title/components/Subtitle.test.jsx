import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import { matcher, serializer } from 'jest-glamor-react'
import SubTitle from './SubTitle'

expect.addSnapshotSerializer(serializer)
expect.extend(matcher)

describe('<SubTitle />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<SubTitle />)
    expect(shallowToJson(wrapper)).toMatchSnapshotWithGlamor()
  })
})
