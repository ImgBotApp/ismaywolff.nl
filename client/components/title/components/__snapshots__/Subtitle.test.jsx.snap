// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SubTitle /> renders correctly 1`] = `
<p
  className="css-2h84iw"
/>
`;
